apps/chat/src/components/Folder/Folder.tsx

```diff
@@ -849,12 +849,20 @@ const Folder = <T extends ConversationInfo | PromptInfo | DialFile>({
               data-qa="folder-name"
             >
               <Tooltip
-                tooltip={t(
-                  getEntityNameError(isNameInvalid, isInvalidPath, isExternal),
-                )}
-                hideTooltip={!isNameOrPathInvalid}
+                tooltip={
+                  currentFolder.name ||
+                  t(
+                    getEntityNameError(
+                      isNameInvalid,
+                      isInvalidPath,
+                      isExternal,
+                    ),
```

Comment:
Error will be always hidden in this case
```suggestion
                  !isNameOrPathInvalid
                  ? currentFolder.name
                  : t(
                    getEntityNameError(
                      isNameInvalid,
                      isInvalidPath,
                      isExternal,
                    ),
```

Reply:
Thanks for your comment, I fixed it.